src/test/scala/com/microsoft/hyperspace/index/ContentTest.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index
+
+import java.nio.file.{Files, Path, Paths}
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.catalyst.plans.SQLHelper
+
+import com.microsoft.hyperspace.TestUtils
+import com.microsoft.hyperspace.util.{JsonUtils, PathUtils}
+
+class ContentTest extends SparkFunSuite with SQLHelper {
+
+  test("test files api") {
+    withTempPath { p =>
+      // Prepare some files and directories.
+      Files.createDirectories(Paths.get(p.getAbsolutePath))
+      val dir = Files.createDirectories(Paths.get(p.getAbsolutePath))
+      Files.createTempFile(dir, "f1", "")
```

Comment:
Can you add a test where you have files in different directory at different level?
```
/tmp/a/1.file
/tmp/a/b/2.file
```
, for example.

Reply:
Thanks @imback82 , done